### PR TITLE
Fix loading for nested through associations

### PR DESCRIPTION
**Pull request by rschef**
Patch to address #55 based on PR #58 by @gbxl

### CODE REVIEW

**Comment by gbxl** [comment]
Ah! Looks like you figured what I was missing! Can't wait to see this merged :). Thanks for you work @rschef !

File: lib/dataloader/ecto.ex [hunk 1]
```diff
@@ -326,7 +326,11 @@ if Code.ensure_loaded?(Ecto) do
       defp chase_down_queryable([field | fields], schema) do
         case schema.__schema__(:association, field) do
           %{queryable: queryable} ->
-            chase_down_queryable(fields, queryable)
+           chase_down_queryable(fields, queryable)
```
Comment (gbxl) [inline lib/dataloader/ecto.ex]: Missing an indent here?

**Comment by benwilson512** [comment]
Thanks!

**Comment by gbxl** [comment]
Thanks @benwilson512, do you have a sense of when the next release could be?

**Pull request merged**